.github/workflows/check.yml

```diff
@@ -0,0 +1,14 @@
+name: Check
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check:
+    uses: wetransform/gha-workflows/.github/workflows/sbt-library-check.yml@master
+    with:
+      java-version: 17
+      expect-tests: false # library does not contain any tests yet
+    secrets: inherit
```

Comment:
`inherit` does not work here because the shared workflow is from another organization, but it is also not needed in this case, so please remove